controllers/databasecluster_controller.go

```diff
@@ -1050,45 +1146,129 @@ func (r *DatabaseClusterReconciler) reconcilePXC(ctx context.Context, req ctrl.R
 	return r.Status().Update(ctx, database)
 }
 
-func (r *DatabaseClusterReconciler) createPGBackrestSecret(
+// Creates or updates the PG Backrest secret
+func CreatePGBackrestSecret(
```

Comment:
Is there any reason the functions removed from `DatabaseClusterReconciler` are now exported? It doesn't seem they need to be, right?

Reply:
I removed the `DatabaseClusterReconciler` and exported them specifically because they are needed in the `DatabaseClusterBackupReconciler` as well.

Reply:
Yes, they're in the same package. No need to export the methods.

Reply:
Yes, you're correct, thanks for spotting it, I'll fix it